xt/lib/Pherkin/Extension/pageobject_steps/nav_steps.pl

```diff
@@ -90,6 +91,11 @@
     'overhead entry' => 'PageObject::App::Parts::Overhead',
     'system defaults' => 'PageObject::App::System::Defaults',
     'system taxes' => 'PageObject::App::System::Taxes',
+    'year-end' => ' PageObject::App::Closing',
+    'Enter Inventory' => ' PageObject::App::Parts::AdjustSetup',
+    'GS Search' => ' PageObject::App::Search::GoodsServices',
+    'RPT Balance Sheet' => ' PageObject::App::Report::Filters::BalanceSheet',
+    'TA Inventory' => ' PageObject::App::Parts::AdjustSearchUnapproved',
```

Comment:
I just followed like the 2 above, which weren't more descriptive. If those are meant to be shown to the user, we really need to have them descriptive.